### PR TITLE
1867 doc for static properties

### PR DESCRIPTION
**Pull request by TheSolY**
Added to the vision properties guide

### CODE REVIEW

File: docs/source/user-guide/vision/vision_properties.rst [hunk 1]
```diff
@@ -32,6 +32,14 @@ Example for specific scenarios in which measuring properties may come in handy:
    the snow, the brightness of the image may be used to predict the label "wolf" easily. In this case, a model
    might not learn to discern wolf from dog by the animal's characteristics, but by using the background color.
 
+Passing Properties to the Checks
+================================
+The properties can be passed to the check in one of two ways:
```
Comment (noamzbr) [inline docs/source/user-guide/vision/vision_properties.rst:37]:
```suggestion
By default, checks using properties will use the :ref:`built-in properties <Deepchecks' Built-in Properties>`. Those default properties can be modified or overridden in one of two ways:
```

File: docs/source/user-guide/vision/vision_properties.rst [hunk 2]
```diff
@@ -32,6 +32,14 @@ Example for specific scenarios in which measuring properties may come in handy:
...
+
+#. **Properties** - a list of functions to be calculated on the data during the check (specific format listed below)
```
Comment (noamzbr) [inline docs/source/user-guide/vision/vision_properties.rst:39]: You can also select a subset of existing properties, no?

File: docs/source/user-guide/vision/vision_properties.rst [hunk 3]
```diff
@@ -32,6 +32,14 @@ Example for specific scenarios in which measuring properties may come in handy:
...
+   passed to the check init.
+#. **Static Properties** - a dictionary with the result of pre calculated properties per sample passed to the check run.
```
Comment (noamzbr) [inline docs/source/user-guide/vision/vision_properties.rst:41]: Should be named "Pre-Calculated Properties" IMO
Maybe a word here on why I'd want that, or alternatively (as this is explained in the relevant section) link to the section, and just way "Properties calculated per sample before running the check. Read more ..."

**Review by noamzbr — CHANGES_REQUESTED** [review]

File: docs/source/user-guide/vision/vision_properties.rst (hunk 2 above)
Comment (TheSolY) [reply docs/source/user-guide/vision/vision_properties.rst:39]: @noamzbr I'm not aware of such option

File: deepchecks/vision/utils/vision_properties.py [hunk 4]
```diff
@@ -130,3 +130,45 @@ def static_prop_to_cache_format(static_props: STATIC_PROPERTIES_FORMAT) -> PROPE
...
     return props_cache
+
+
+def static_properties_from_df(df,
+                              image_cols: Tuple = (),
+                              label_cols: Tuple = (),
+                              prediction_cols: Tuple = (),
+                              partial_image_cols: Tuple = ()) -> STATIC_PROPERTIES_FORMAT:
+    """
+    Transform the precalculated properties to the static properties format.
```
Comment (noamzbr) [inline deepchecks/vision/utils/vision_properties.py:141]:
```suggestion
    Transform the pre-calculated properties from a DataFrame to the expected dict format.
    
    Read more about the excepted dict format for pre-calculated properties at X
```
Replace X with link to your docs

File: docs/source/user-guide/vision/vision_properties.rst [hunk 5]
```diff
@@ -119,6 +119,18 @@ other hand for label & predictions we allow each one to have multiple primitive
 box), which means the returned list may contain either primitives values or a lists of primitive values per
 label/prediction.
 
+
+Customizing the Checks' Properties
+==================================
+By default, checks using properties will use the :ref:`built-in properties <Deepchecks' Built-in Properties>`.
+Those default properties can be modified or overridden in one of two ways:
```
Comment (noamzbr) [inline docs/source/user-guide/vision/vision_properties.rst:126]:
```suggestion
Those default properties can be overridden in one of two ways:
```
If we can't select a subset...

**Review by noamzbr — APPROVED** [review]

**Pull request merged**